Gemfile

```diff
@@ -2,3 +2,5 @@
 
 source "https://rubygems.org"
 gemspec
+
+gem 'jekyll-seo-tag'
```

Comment:
For adding of new Gems, add it in the `tale.gemspec` file instead. Also add it in the `_config.yml` file under the `plugins` section

Reply:
Done!